### PR TITLE
Explain how to write websocket adapters

### PR DESCRIPTION
**Pull request by OrangeTux**

### CODE REVIEW

**Comment by proelke** [comment]
I would consider adding your full example from #179 into the examples folder as well.

**Comment by laysauchoa** [comment]
I think is a good idea to have a full example, where one could just use to run and go from there.

File: README.rst [hunk 1]
```diff
@@ -163,6 +163,45 @@ Charge point
     if __name__ == '__main__':
        asyncio.run(main())
 
+
+Websocket support
+-----------------
+
+Although this library depends on websocket support, no specific implementation
```
[the remark this answers is not in the document]
Comment (OrangeTux) [reply README.rst:170]: Correct. I try to distinguish between websockets the standard and `websockets` the Python implementation by wrapping the latter in backticks ```.

**Comment by Phantomo** [comment]
Thank you for this example and I think will be cool if experience devs add examples for popular frameworks and libs like a tornado, aiohttp, etc.

**Comment by OrangeTux** [comment]
Closed due inactivity.

**Pull request closed without merging**